app/src/test/js/searchKeysCtrlTest.js

```diff
@@ -0,0 +1,48 @@
+describe('SearchKeysCtrl', function() {
+
+    beforeEach(module('esbMessageAdminApp', function($provide) {
+        $provide.value("EsbMessageService", mockService);
+    }));
+
+    var searchKeysCtrl, rootScope;
+
+    beforeEach(inject(function($rootScope, $controller, _$q_) {
+        rootScope = $rootScope;
+        searchKeysScope = $rootScope.$new();
+        $q = _$q_;
+        spyOn(mockService, "getSearchKeysTree").and.returnValue($q
+                .when(searchKeysSuccessResponse));
+        searchKeysCtrl = $controller('SearchKeysCtrl', {
+            '$scope' : searchKeysScope
+        });
+        rootScope.$apply();
+
+    }));
+
+    it('loads Tab', function() {
```

Comment:
"Loads tab" is still so vague. Why not, `it('sets scope's parent and entities properties to search keys tree')`? From an outsider looking in, it "loads tab" tells me nothing about what the responsibility of "loading the tab" actually consists of. Frankly you could even break down that into two tests: `it("set's scope.parent to search keys tree on load")` and `it("set's scope.entities to search keys tree on load")`